libs/dotnet-sdk/FeatureBoardHttpClient.cs

```diff
@@ -33,14 +37,34 @@ public FeatureBoardHttpClient(HttpClient httpClient, LastETagProvider lastModifi
     if (null != eTag)
       request.Headers.IfNoneMatch.Add(eTag);
 
+    var retryAfter = _retryAfter();
+    if (retryAfter != null && retryAfter.Date > DateTimeOffset.Now)
+    {
+      // Do not call API
+      _logger.LogWarning("Failed to get latest flags. Service returned error 429 (Too Many Requests). Retry after: {retryAfter}", retryAfter.Date.Value.ToString());
```

Comment:
I think this should be an info level message

Reply:
I think a warning is correct as the dev has setup the config incorrectly